homeassistant/components/sensor/melissa.py

```diff
@@ -0,0 +1,99 @@
+"""
+Support for Melissa climate Sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.melissa/
+"""
+import logging
+
+from homeassistant.components.melissa import DOMAIN, DATA_MELISSA
+from homeassistant.const import TEMP_CELSIUS, STATE_UNKNOWN
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = [DOMAIN]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the melissa sensor platform."""
+    sensors = []
+    connection = hass.data[DATA_MELISSA]
+    devices = connection.fetch_devices().values()
```

Comment:
Is `fetch_devices()` guaranteed to return an empty dict(?) if something fails, or should here be a check before trying to access its values?

Reply:
It always return a dict